reflex/.templates/jinja/custom_components/src.py.jinja2

```diff
@@ -0,0 +1,59 @@
+"""Reflex custom component {{ component_class_name }}."""
+import reflex as rx
+
+from reflex.vars import Var
+from typing import Any
+
+
+# Some libraries you may want to wrap may require dynamic imports.
+# This is because they they may not be compatible with Server-Side Rendering (SSR).
+# To handle this in Reflex all you need to do is subclass NoSSRComponent instead.
+# For example:
+# from reflex.components.component import NoSSRComponent
+# class {{ component_class_name }}(NoSSRComponent):
+#     pass
+
+
+class {{ component_class_name }}(rx.Component):
+    """{{ component_class_name }} component."""
+
+    # The React library to wrap.
+    library = "Fill-Me"
+
+    # The React component tag.
+    tag = "Fill-Me"
+
+    # If the tag is the default export from the module, you can set is_default = True.
```

Comment:
"you can set" should change to "you must set" as otherwise it throws a server side error